stack/base/Dockerfile

```diff
@@ -17,10 +17,11 @@ WORKDIR /opt/
 ARG AIIDA_VERSION
 
 # Install the shared requirements.
+# XXX: bring back mamba-bash-completion once it is available on conda-forge.
+# https://github.com/conda-forge/mamba-bash-completion-feedstock/pull/2
```

Comment:
@danielhollas, here is the workaround required for the `mamba-bash-completion` which not supported in arm64.